### PR TITLE
add the ability to set slide theme in slide options

### PR DESCRIPTION
**Pull request by butlerx**
for #437

### CODE REVIEW

**Comment by jackycute** [comment]
Thanks @butlerx, this looks great!

But I think we need to check if the theme is exist in the reveal.js.
https://github.com/hakimel/reveal.js/tree/master/css/theme

**Comment by butlerx** [comment]
ive added a check there i wasnt really sure if doing a relative path from lib or the root dir is better though so i chose the root dir

**Comment by jackycute** [comment]
@butlerx I think you did a great job.
Let's review.

File: lib/utils.js [hunk 1]
```diff
@@ -23,3 +25,10 @@ exports.getImageMimeType = function getImageMimeType (imagePath) {
       return undefined
   }
 }
+
+exports.isRevealTheme = function isRevealTheme (theme) {
+  if (fs.existsSync(path.join(process.cwd(), 'public', 'build', 'reveal.js', 'css', 'theme', theme + '.css'))) {
```
Comment (jackycute) [inline lib/utils.js]: We should use `__dirname` instead of process.cwd() to prevent getting wrong path.

Comment (butlerx) [inline lib/utils.js]: `__dirname` would give `/home/user/hackmd/lib/` happy to switch  to it itll just have to be `__dirname, '..'`

Comment (jackycute) [inline lib/utils.js]: It's ok to use `..` here, lots better now.

**Comment by jackycute** [comment]
Thanks @butlerx

**Pull request merged**